compiler/quilt/tools/command.py

```diff
@@ -1340,6 +1340,14 @@ def list_users(team=None):
     resp = session.get('%s/api/users/list' % url)
     return resp.json()
 
+def list_users_detailed(team=None):
```

Comment:
You need to add this to the export list in __init__.py for it to be available in the API.